packages/database/src/getConnectionConfig.js

```diff
@@ -3,27 +3,45 @@
  * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
  */
 
-// Note: Must use function to guarantee environment variables have loaded.
+const CI_CONNECTION_CONFIG = {
+  host: process.env.CI_TEST_DB_URL,
+  user: process.env.CI_TEST_DB_USER,
+  password: process.env.CI_TEST_DB_PASSWORD,
+  database: process.env.CI_TEST_DB_NAME,
+  ssl: null,
+};
+
+const SERVER_CONNECTION_CONFIG = {
+  host: process.env.DB_URL,
+  user: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_NAME,
+  ssl:
+    process.env.DB_ENABLE_SSL === 'true'
+      ? {
+          // Test server cannot turn on ssl, so sets the env to disable it
+          rejectUnauthorized: false,
+        }
+      : null,
+};
+
+const validateConfig = config => {
+  if (config.ssl) {
+    return;
+  }
+
```

Comment:
```suggestion
```
If ssl is set, it doesn't guarantee we have the other variables set up correctly (and we still need to)

Reply:
I wasn't sure about that and actually I should have left a question-comment! Thanks for catching this!